python/mxnet/libinfo.py

```diff
@@ -29,6 +29,9 @@ def find_lib_path():
         dll_path.extend([p.strip() for p in os.environ['LD_LIBRARY_PATH'].split(":")])
     if os.name == 'nt':
         dll_path = [os.path.join(p, 'libmxnet.dll') for p in dll_path]
+    elif platform.system() == 'Darwin':
+        dll_path = [os.path.join(p, 'libmxnet.dylib') for p in dll_path]+ \
```

Comment:
Got it. I think this is fine then, as long as both .dylib and .so are being searched.